packages/ui-components/src/components/calendar-day/calendar-day.tsx

```diff
@@ -1,53 +1,53 @@
-import { Component, Event, EventEmitter, h, Host, Prop } from '@stencil/core';
-import { ICalendarDay, ICalendarDayEvents, IClickDayEvent, IMouseEnterEvent, IMouseLeaveEvent } from './calendar-day.types';
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+import { ICalendarDay, ICalendarDayEvents } from './calendar-day.types';
 
 /**
  * @part day-container - The day button container.
  */
 @Component({
 	tag: 'kv-calendar-day',
-	styleUrls: {
-		night: 'calendar-day.night.scss',
-		light: 'calendar-day.light.scss'
-	},
+	styleUrl: 'calendar-day.scss',
 	shadow: true
 })
 export class KvCalendarDay implements ICalendarDay, ICalendarDayEvents {
 	/** @inheritdoc */
-	@Prop({ reflect: true }) day: number;
+	@Prop({ reflect: true }) day!: number;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) disabled: boolean = false;
+	@Prop({ reflect: true }) disabled?: boolean = false;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) active: boolean = false;
+	@Prop({ reflect: true }) active?: boolean = false;
 	/** @inheritdoc */
-	@Prop({ reflect: false }) inRange: boolean = false;
+	@Prop({ reflect: false }) inRange?: boolean = false;
 	/** @inheritdoc */
-	@Prop({ reflect: false }) leftRounded: boolean = false;
+	@Prop({ reflect: false }) isRangeStartDate?: boolean = false;
 	/** @inheritdoc */
-	@Prop({ reflect: false }) rightRounded: boolean = false;
+	@Prop({ reflect: false }) isRangeEndDate?: boolean = false;
+	/** @inheritdoc */
+	@Prop({ reflect: false }) isToday?: boolean = false;
+	/** @inheritdoc */
+	@Prop({ reflect: false }) isBetweenSelectedDates?: boolean = false;
```

Comment:
Isn't this the combination of `isRangeStartDate`, `isRangeEndDate` and `inRange`?

Reply:
No, the inRange represents the dates when we are hovering. In this case we have the initial date selected and we are hovering the days which should represent a certain state. 
The isBetweenSelectedDates represents the state where the start and end date are selected and we set the days in between with another style, which is not the same as the start or end date.